specification/security/resource-manager/readme.md

```diff
@@ -65,4 +74,31 @@ csharp:
   payload-flattening-threshold: 2
   output-folder: $(csharp-sdks-folder)/Security/Management.Security/Generated
   clear-output-folder: true
-```
\ No newline at end of file
+```
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: mysql
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-composite-v1
+```
+
+### Tag: package-composite-v1 and go
+
+These settings apply only when `--tag=package-composite-v1 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-composite-v1' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/security/mgmt/2017-08-01-preview/security

```

Comment:
If you fix the namespace value you can replace the hard-coded ```security``` path/namespace with ```$(namespace)```.  One day I'd like to see the API version be defined along with a tag so we don't have to duplicate it here.